packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -76,6 +76,10 @@ function NavigationMenuSelector(
 		return null;
 	}
 
+	const navigationStorageMap = new Map(
```

Comment:
Similar nit, let's refer to classic menus, as "navigation" hopefully is about block menus. So maybe name the variable `importedClassicMenus`? I'd also leave out "type in name" as it's not common in this codebase.

Reply:
Done!